### ISSUE TITLE
Sharing GIFs from other apps doesn't work

### ISSUE THREAD

**Issue by AlexTalker** [TelegramMessenger/Telegram-iOS#411]
### Checklist
- [x] I am reporting an issue in existing functionality that does not work as intended
- [x] I've searched for existing [GitHub issues](https://github.com/telegrammessenger/Telegram-iOS/issues)

### Description
I am trying to send any GIF from a post in VK application to a contact in Telegram

### Expected Behavior
I successfully can share a GIF from VK via native share interface of iOS

### Actual Behavior
GIF is shown as successfully shared during the process
but peeking into the app reveals that it cannot be sent and the user offered 2 options:
1. Try the resend action
2. Remove the message(i.e. the GIF)

Trying 1st one yields that it cannot be send and the options are presented once again.
Deletion works successfully thou.

For comparison, I tried to send the same GIF into WhatsApp and it just worked.
Thus I suspect it's not an iOS-release bug, nor VK's bug.

### Steps to Reproduce
1. Find(in latest VK app) any post with a GIF
2. Press share button, this will show VK's recipient's list
3. Press button to share into other apps(native iOS method) in upper left corner
4. Select Telegram
5. In Telegram, select any contact(including messaging to one's self!)
6. Press "Share" button
7. Go into the Telegram app and observe result

### Screenshots and Videos 
I was a bit lame on this part,
thou I think words described the situation precisely enough.

### Environment
**Device:** `iPhone SE(1st generation)`

**iOS version**: `14.0.1`

**App version:** `7.2`

**Comment by AlexTalker** [comment]
Just a side note, this is happening not the first time and so got me finally annoyed enough to fill in the report.

**Comment by atineo** [comment]
This is happening in any app that lets you share the gif, reddit app (Apollo) for example.

When you share as gif from the app you get the exclamation mark in Telegram, and if you share it again inside Telegram to another contact or the same contact you get the gif working fine.

### Environment
**Device**: `iPhone 12 Pro Max`

**iOS version**: `14.2.1`

**App version**: `7.2.1`

**Comment by AlexTalker** [comment]
@atineo So you're suggesting if I sent it to myself("Saved messages") and get the error, I can then redirect it to target contact and it will work out?
Or you mean it works if done twice?

**Comment by atineo** [comment]
> @atineo So you're suggesting if I sent it to myself("Saved messages") and get the error, I can then redirect it to target contact and it will work out?
> Or you mean it works if done twice?

Yeah you can send it to "Saved Messages", then the error will happen and then share from there to another contact and it will work

**Comment by AlexTalker** [comment]
Sounds a bit mad, considering retry button did nothing to resolve the issue(!!!) :rofl:

**Comment by Banck** [comment]
Hey!
We've faced a similar issue. In our iOS app we give the opportunity to share gif via UIActivityViewController, but when gif was shared to telegram there is an error in the telegram chat.

**Comment by Banck** [comment]
Update: We've checked telegram iOS client v.6.2.1 and all is fine. Seems only on the latest releases gif sharing doesn't work via UIActivityController

**Comment by AlexTalker** [comment]
@Banck Thanks for the details.
I personally feel itchy to get this fixed if only I had OS X but VPS pricing is like 30$ a month or higher, which feels a bit expensive to me :rofl: 
Guess just gonna wait till someone else do it.
